### PR TITLE
Add more items to guidelines Code Standards.

### PR DESCRIPTION
**Pull request by kersommoura**
Add more non mandatory items to existing Code Standards.

In order to make clear in the documentation what expect from a
contributor, add few more details about code standard.

### CODE REVIEW

**Comment by kersommoura** [comment]
@Ichimonji10, I was thinking about something simple like that. Any other suggestion can be added at same section later. Feel free to change.

File: docs/about.rst [hunk 1]
```diff
@@ -99,6 +99,13 @@ Please adhere to the following guidelines:
   reason. As another example, do not add a test that makes dozens of concurrent
   requests to a public service such as docker hub.
 
+These next guidelines **are not** mandatory, but will match the formatting adopted
+in the existing code base.
+
+* Docstrings should not end with a blank line.
+* Docstrings should be line-wrapped at 79 characters.
```
Comment (Ichimonji10) [inline docs/about.rst]: More generally, all code should be line-wrapped at 79 characters.

Comment (kersommoura) [reply docs/about.rst]: What about the following statement?

Line length should be line-wrapped at 79 characters.

Comment (Ichimonji10) [reply docs/about.rst]: Sounds great.

File: docs/about.rst [hunk 2]
```diff
@@ -99,6 +99,13 @@ Please adhere to the following guidelines:
   reason. As another example, do not add a test that makes dozens of concurrent
   requests to a public service such as docker hub.
 
+These next guidelines **are not** mandatory, but will match the formatting adopted
+in the existing code base.
+
+* Docstrings should not end with a blank line.
+* Line length should be line-wrapped at 79 characters.
```
Comment (Ichimonji10) [inline docs/about.rst]: I just re-read this and realized that it sounds a bit awkward. Can it read "lines should be hard wrapped at 79 characters?" Please and thank you.

Comment (kersommoura) [reply docs/about.rst]: Done.

**Pull request merged**